chicago/models.py

```diff
@@ -133,11 +133,34 @@ def full_text_doc_url(self):
             return None
 
 
+class ChicagoOrganization(Organization):
+    class Meta:
+        proxy = True
+        app_label = "chicago"
+
+    @property
+    def recent_events(self):
+        # need to look up event participants by name
+        events = ChicagoEvent.objects.filter(
+            participants__entity_type="organization", participants__name=self.name
+        )
+        events = events.order_by("-start_date").all()
+        return events
+
+
 class ChicagoEvent(Event):
     class Meta:
         proxy = True
         app_label = "chicago"
 
+    @property
+    def attendance(self):
```

Comment:
how how you are using this it would be fine just have this be

```python
def attendees(self):
   return self.participants.filter(entity_type="person")
``` 

don't need the set here.

additionally, would be good to use this method in the event detail view `get_context_data` method.